src/dialects/postgres/query-generator.js

```diff
@@ -126,12 +126,12 @@ export class PostgresQueryGenerator extends AbstractQueryGenerator {
 
   /**
    *
-   * @param {*} _       Unused variable (database name)
-   * @param {object}    [options] Options
-   * @param {string}    [options.schema] Schema name to search for (defaults to 'public')
-   * @returns {string}  SQL statement to show all tables, including tableName and schema
+   * @param {*} _database Unused variable (database name)
+   * @param {object}      [options] Options
+   * @param {string}      [options.schema] Schema name to search for (defaults to 'public')
+   * @returns {string}    SQL statement to show all tables, including tableName and schema
    */
-  showTablesQuery(_, options) {
+  showTablesQuery(_database, options) {
```

Comment:
I'd rather avoid complexifying our already complex API: the first parameter and the second parameter do the same thing, it was just incorrectly named 'database' in mariadb because it's the same thing in that dialect

Consider one of the two interfaces instead:

```typescript
showTablesQuery(schema?: string): string;
// or
showTablesQuery(options?: { schema?: string }): string;
```